src/glibc/sysdeps/unix/sysv/linux/epoll_ctl.c

```diff
@@ -20,13 +20,14 @@
 #include <sysdep.h>
 #include <syscall-template.h>
 #include <lind_syscall_num.h>
+#include <addr_translation.h>
 
 libc_hidden_proto (epoll_ctl)
 
 int
 epoll_ctl (int __epfd, int __op, int __fd,
```

Comment:
epoll_event is a weird struct here, we probably need to convert that here manually